config/locales/views/extensions/en.yml

```diff
@@ -1,6 +1,10 @@
 ---
 en:
   extensions:
+    api:
+      extension_empty: The extension does not exist
+      extension_exist: The extension already exist
+      time_delta_empty: Time delta needs to be provided
```

Comment:
As a I noted above, rather than reporting this custom error message, you can make use of the default validation message in this case